### PR TITLE
feat/PRO-2573/token-atlas-app

### PR DESCRIPTION
**Pull request by RanaBug**
## Description

- UI design implementation of the Token Atlas app with dummy data (no API integration yet)

## How Has This Been Tested?

- Manual testing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/apps/token-atlas/components/ChainCard/ChainCard.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import LogoEx from '../../images/token-atlas-logo-small.svg';
+
+// components
+import Body from '../Typography/Body';
+
+const ChainCard = () => {
+    return (
+        <div className="flex rounded-[50px] bg-medium_grey p-1 pr-3 items-center h-8 max-w-[150px]">
+            <img src={LogoEx} className="w-[24px] h-[24px] object-fill rounded-full mr-2" />
+            <Body className='truncate'>EthereumEthereumEthereum</Body>
```
Comment (IAmKio) [inline src/apps/token-atlas/components/ChainCard/ChainCard.tsx:11]: Is that supposed to be there? 😅

File: src/apps/token-atlas/components/HeaderSearch/HeaderSeach.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import { setIsSearchTokenModalOpen } from '../../reducer/tokenAtlasSlice';
+
+// images
+import TokenAtlasLogo from '../../images/token-atlas-logo-small.svg';
+
+// components
+import Body from '../../components/Typography/Body';
+import SearchLogo from '../../images/circle-search.svg';
+
+const HearderSearch = () => {
```
Comment (IAmKio) [inline src/apps/token-atlas/components/HeaderSearch/HeaderSeach.tsx:14]:
```suggestion
const HeaderSearch = () => {
```

File: src/apps/token-atlas/components/TokenGraphColumn/TokenGraphColumn.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from 'react';
+
+// images
+import LogoEx from '../../images/token-atlas-logo-small.svg';
+import ArrowGreen from '../../images/arrow-circle-green.svg';
+// import ArrowRed from '../../images/arrow-circle-red.svg';
+// import ArrowRedSmall from '../../images/arrow-circle-red-small.svg';
```
Comment (IAmKio) [inline src/apps/token-atlas/components/TokenGraphColumn/TokenGraphColumn.tsx:7]: Just a reminder to remove unless you may need it in future

File: src/apps/token-atlas/icon.png
Comment (IAmKio) [inline src/apps/token-atlas/icon.png:1]: Assuming here that we don't have the high res file which is why this is here for now

**Review by IAmKio — COMMENTED** [review]
Just some minor comments - but this can be approved as soon as we just check the comments below

File: src/apps/token-atlas/components/ChainCard/ChainCard.tsx (hunk 1 above)
Comment (RanaBug) [reply src/apps/token-atlas/components/ChainCard/ChainCard.tsx:11]: No, well it is a dummy data (placeholder), the reason I put 3 times Ethereum is to test when the text is too long haha

File: src/apps/token-atlas/components/HeaderSearch/HeaderSeach.tsx (hunk 2 above)
Comment (RanaBug) [reply src/apps/token-atlas/components/HeaderSearch/HeaderSeach.tsx:14]: Oopsy,  done

File: src/apps/token-atlas/components/TokenGraphColumn/TokenGraphColumn.tsx (hunk 3 above)
Comment (RanaBug) [reply src/apps/token-atlas/components/TokenGraphColumn/TokenGraphColumn.tsx:7]: I will need in the future, need the data to be dynamic to fully implement those

File: src/apps/token-atlas/icon.png
Comment (RanaBug) [reply src/apps/token-atlas/icon.png:1]: yes exactly

**Review by IAmKio — APPROVED** [review]
Great work! 🚀

**Pull request merged**